dm/ctl/common/config.go

```diff
@@ -106,7 +115,11 @@ func (c *Config) Parse(arguments []string) error {
 		return errors.Annotatef(err, "specify master addr %s", c.MasterAddr)
 	}
 
-	c.adjust()
+	err = c.adjust()
+	if err != nil {
+		return errors.Trace(err)
```

Comment:
Since we are using the latest version of `pingcap/errors`, we don't need to check for `nil` before calling `errors.Trace`.

Reply:
Related PR in `pingcap/errors`: https://github.com/pingcap/errors/pull/12